src/Notes/NoteInitializer.php

```diff
@@ -70,7 +59,9 @@ public function register(): void {
 	 * Loop through all notes to add any that should be added.
 	 */
 	public function add_notes(): void {
-		foreach ( $this->notes as $note ) {
+		$notes = $this->container->get( Note::class );
```

Comment:
In other classes we have added [a comment at the top](https://github.com/woocommerce/google-listings-and-ads/blob/1.11.1/src/MerchantCenter/MerchantCenterService.php#L34-L43) showing which classes we retrieve from the container. This makes it easier to see at a glance which dependencies are required for the class.